release/src/main/scripts/build_release_candidate.sh

```diff
@@ -45,6 +45,9 @@ PYTHON_ARTIFACTS_DIR=python
 BEAM_ROOT_DIR=beam
 WEBSITE_ROOT_DIR=beam-site
 
+DOCKER_IMAGE_DEFAULT_REPO_ROOT=apache
+DOCKER_IMAGE_DEFAULT_REPO_PREFIX=beam_
```

Comment:
Would it be worth sourcing these from a script?

Reply:
Yeap, that's a great idea and I think it's better to have a script which holds all variables and shared by all release scripts, so I haven't used a script this time. I created a ticket to fix it. [BEAM-9207](https://issues.apache.org/jira/browse/BEAM-9207)